scripts/buildTools.ts

```diff
@@ -286,10 +289,13 @@ async function getDumps(connection: Client) {
     await retrieve(connection, resp, "dumps");
 }
 
-async function artifacts(connection: Client) {
+async function artifacts(connection: Client, packageApf: boolean) {
     const artifactPaths = ["c/zowex", "golang/zowed"];
+    if (packageApf) {
+        artifactPaths.push("c/zoweax");
```

Comment:
You may have already discussed this, but don't users have to `extattr +ap zoweax` after uploading/deploying/installing?

Reply:
@dkelosky Do you know the answer to this? Since we already have some folks internally adopting it 😋